include/ccf/node_startup_state.h

```diff
@@ -14,8 +14,7 @@ namespace ccf
     partOfPublicNetwork,
     partOfNetwork,
     readingPublicLedger,
-    readingPrivateLedger,
-    verifyingSnapshot
+    readingPrivateLedger
```

Comment:
That enum is never serialized to the ledger, right?

Reply:
It isn't. It's only reported by one of the `/node` operator endpoints.

Reply:
Great, I thought this was the case, just wanted to confirm.